lib/main.js

```diff
@@ -0,0 +1,110 @@
+'use babel'
+
+import Path from 'path'
+import {CompositeDisposable} from 'atom'
+import {spawnWorker} from './helpers'
+import escapeHTML from 'escape-html'
+
+export default {
+  config: {
+    lintHtmlFiles: {
+      title: 'Lint HTML Files',
+      description: 'You should also add `eslint-plugin-html` to your .eslintrc plugins',
+      type: 'boolean',
+      default: false
+    },
+    useGlobalEslint: {
+      title: 'Use global ESLint installation',
+      description: 'Make sure you have it in your $PATH',
+      type: 'boolean',
+      default: false
+    },
+    showRuleIdInMessage: {
+      title: 'Show Rule ID in Messages',
+      type: 'boolean',
+      default: true
+    },
+    disableWhenNoEslintrcFileInPath: {
+      title: 'Disable when no .eslintrc is found',
+      type: 'boolean',
+      default: true
+    }
+  },
+  activate: function() {
+    require('atom-package-deps').install('linter-eslint')
```

Comment:
No need to specify the name here with the current version of `atom-package-deps`.